peekingduck/pipeline/nodes/draw/group_bbox_and_tag.py

```diff
@@ -26,7 +26,30 @@
 
 class Node(AbstractNode):
     """This node draws large bounding boxes over multiple object bounding boxes
-    which have been identified as belonging to large groups."""
+    which have been identified as belonging to large groups.
+
+    The draw group_bbox_and_tage node uses the obj_groups and large_groups from the
+    heuristic predictions to draw group bboxes and the large group message tag onto the image.
+    For better understanding of the usecase, refer to the group size checking usecase.
+
+    Inputs:
+
+        |img|
+
+        |bboxes|
+
+        |obj_groups|
+
+        |large_groups|
+
+    Outputs:
+        |none|
+
+    Configs:
+        tag (:obj:`str`):
+            string message printed in the case of a large group detected.
+            Default message is "LARGE GROUP!"
```

Comment:
declare defaults as such:

https://github.com/aimakerspace/PeekingDuck/blob/313e530458ec6e3f1203518d0aab88d45b1277a0/peekingduck/pipeline/nodes/output/csv_writer.py#L41

Reply:
Missed out changing this. Corrected